### ISSUE TITLE
Remove references to tudat-multidoc

### ISSUE THREAD

**Issue by yiquintero** [tudat-team/tudatpy-feedstock#21]
## Description

As described in [issue 150](https://github.com/tudat-team/tudatpy/issues/150), of the tudatpy repository, [tudat-multidoc](https://github.com/tudat-team/tudat-multidoc) and it's functionality are being phased-out in favour of a simpler way to generate the [tudatpy API docs](https://py.api.tudat.space/en/latest/). For this reason, all references to multidoc must be removed from this repository.

## Solution
Remove [lines 6-11 in recipe/build.sh](https://github.com/tudat-team/tudatpy-feedstock/blob/1b80797a3306ec1336ce6258444aa9997e9bfc2c/recipe/build.sh#L6). These lines refer to a file used as part of the tudat-multidoc pipelines. 

At the moment, the lines need to be removed **only from the develop branch** since tudatpy's master branch is still using the multidoc logic. Once tudatpy's develop branch is merged into master, the aforementioned lines can be removed from the master branch as well.

Note: it is safe to remove these lines as its removal was tested by commenting-them out and making sure the Azure pipelines ran successfully and the online API docs were rendered without issues.
